armi/reactor/converters/tests/test_uniformMesh.py

```diff
@@ -37,7 +37,8 @@ class TestUniformMeshComponents(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        random.seed(987324987234)  # so it's always the same
+        # TODO: WHY would this be necessary? Are our uniform meshes random???
```

Comment:
I believe a unit test does (or used to) set up a random mesh and run this for it. This seed allows the random mesh to always be the same random mesh. could also just hard-code the random mesh.

Reply:
I hadn't actually tried to track this down. I see now this is used in `applyNonUniformHeightDistribution()`.  So I can document that in a comment, yeah. Good call.